### ISSUE TITLE
use arakoon local collapse instead of remote collapse

### ISSUE THREAD

**Issue by jeroenmaelbrancke** [openvstorage/framework#1025]
It would be better to trigger a local collapse of the Arakoons.

1) local collapse will use another process to collapse --> no extra load on the running Arakoon process and will not trigger nomasterlookup issues

2) in the remote collapse we received an error failure("success"), restarting the Arakoon process helps to solve this issue... @toolslive is already checking the issue

@matthiasdeblock can you add your logging to this ticket?

**Comment by matthiasdeblock** [comment]
ovs-workers:

```
2016-10-12 12:10:02 60200 +0200 - pocops-ovs02 - 8841/140686183761728 - lib/scheduled tasks - 161510 - INFO -   Collapsing cluster ovsdb
2016-10-12 12:10:02 60500 +0200 - pocops-ovs02 - 8841/140686183761728 - lib/scheduled tasks - 161511 - INFO -     Collapsing node: a1d5Wg36FMhesTmG
2016-10-12 12:10:02 81900 +0200 - pocops-ovs02 - 8841/140686183761728 - lib/scheduled tasks - 161512 - INFO -     Collapsing node: 0lyCLx1w0S2Nn2Sn
2016-10-12 12:10:02 90000 +0200 - pocops-ovs02 - 8841/140686183761728 - lib/scheduled tasks - 161513 - ERROR - Error during collapsing cluster ovsdb node 0lyCLx1w0S2Nn2Sn
Traceback (most recent call last):
  File "/opt/OpenvStorage/ovs/lib/scheduledtask.py", line 410, in collapse_arakoon
    client.collapse(str(node), 2)
  File "<update_argspec>", line 5, in collapse
  File "/opt/OpenvStorage/ovs/extensions/db/arakoon/pyrakoon/pyrakoon/compat.py", line 152, in wrapped
    raise new_exception
ArakoonException: unknown failure
2016-10-12 12:10:02 90000 +0200 - pocops-ovs02 - 8841/140686183761728 - lib/scheduled tasks - 161514 - INFO -     Collapsing node: OhyeLjFiOeaKF82D
2016-10-12 12:10:03 09700 +0200 - pocops-ovs02 - 8841/140686183761728 - lib/scheduled tasks - 161515 - INFO - Arakoon collapse finished
```

Executing collapse-remote:

```
root@pocops-ovs02:~# arakoon --collapse-remote backendje-nsm_0 172.19.21.21 26406 2
arakoon: main: remote_collapsing_failed: (Failure "unknown failure (EC: 255)")
arakoon: main: backtrace:
arakoon: main: Raised at file "map.ml", line 122, characters 16-25
arakoon: main: Called from file "src/core/lwt.ml", line 161, characters 4-40
root@pocops-ovs02:~# restart ovs-arakoon-backendje-nsm_0 
ovs-arakoon-backendje-nsm_0 start/running, process 3042
root@pocops-ovs02:~# arakoon --collapse-remote backendje-nsm_0 172.19.21.21 26406 2
```

**Comment by wimpers** [comment]
Was also requested by one of the main stakeholders.

**Comment by khenderick** [comment]
Fixed by #1030, packaged in openvstorage-2.7.4-rev.4132.83baaea

**Comment by JeffreyDevloo** [comment]
### Steps
- Setup environment
- Configured environment
- Let it run for a while

### Output

```
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 20200 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1803 - INFO -   Collapsing cluster mybackend-global-abm on 10.100.199.153 completed
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 20300 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1804 - DEBUG -   Collapsing cluster voldrv on 10.100.199.153
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 40400 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1805 - INFO -   Collapsing cluster voldrv on 10.100.199.153 completed
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 40400 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1806 - DEBUG -   Collapsing cluster mybackend02-abm on 10.100.199.153
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 60500 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1807 - INFO -   Collapsing cluster mybackend02-abm on 10.100.199.153 completed
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 60500 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1808 - DEBUG -   Collapsing cluster mybackend-global-nsm_0 on 10.100.199.153
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 79800 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1809 - INFO -   Collapsing cluster mybackend-global-nsm_0 on 10.100.199.153 completed
Nov 16 15:10:02 ovs-node-1 celery[2154]: 2016-11-16 15:10:02 79800 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1810 - DEBUG -   Collapsing cluster mybackend02-nsm_0 on 10.100.199.153
Nov 16 15:10:03 ovs-node-1 celery[2154]: 2016-11-16 15:10:03 00500 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1811 - INFO -   Collapsing cluster mybackend02-nsm_0 on 10.100.199.153 completed
Nov 16 15:10:03 ovs-node-1 celery[2154]: 2016-11-16 15:10:03 00500 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1812 - DEBUG -   Collapsing cluster ovsdb on 10.100.199.153
Nov 16 15:10:03 ovs-node-1 celery[2154]: 2016-11-16 15:10:03 24500 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1813 - INFO -   Collapsing cluster ovsdb on 10.100.199.153 completed
Nov 16 15:10:03 ovs-node-1 celery[2154]: 2016-11-16 15:10:03 73500 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1814 - DEBUG -   Collapsing cluster cacc on 10.100.199.152
Nov 16 15:10:03 ovs-node-1 celery[2154]: 2016-11-16 15:10:03 90500 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1815 - INFO -   Collapsing cluster cacc on 10.100.199.152 completed
Nov 16 15:10:03 ovs-node-1 celery[2154]: 2016-11-16 15:10:03 90600 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1816 - DEBUG -   Collapsing cluster mybackend-global-abm on 10.100.199.152
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 07100 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1817 - INFO -   Collapsing cluster mybackend-global-abm on 10.100.199.152 completed
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 07100 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1818 - DEBUG -   Collapsing cluster voldrv on 10.100.199.152
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 31800 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1819 - INFO -   Collapsing cluster voldrv on 10.100.199.152 completed
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 31800 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1820 - DEBUG -   Collapsing cluster mybackend02-abm on 10.100.199.152
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 60000 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1821 - INFO -   Collapsing cluster mybackend02-abm on 10.100.199.152 completed
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 60100 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1822 - DEBUG -   Collapsing cluster mybackend-global-nsm_0 on 10.100.199.152
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 82200 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1823 - INFO -   Collapsing cluster mybackend-global-nsm_0 on 10.100.199.152 completed
Nov 16 15:10:04 ovs-node-1 celery[2154]: 2016-11-16 15:10:04 82200 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1824 - DEBUG -   Collapsing cluster mybackend02-nsm_0 on 10.100.199.152
Nov 16 15:10:05 ovs-node-1 celery[2154]: 2016-11-16 15:10:05 09300 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1825 - INFO -   Collapsing cluster mybackend02-nsm_0 on 10.100.199.152 completed
Nov 16 15:10:05 ovs-node-1 celery[2154]: 2016-11-16 15:10:05 09400 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1826 - DEBUG -   Collapsing cluster ovsdb on 10.100.199.152
Nov 16 15:10:05 ovs-node-1 celery[2154]: 2016-11-16 15:10:05 33600 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1827 - INFO -   Collapsing cluster ovsdb on 10.100.199.152 completed
Nov 16 15:10:05 ovs-node-1 celery[2154]: 2016-11-16 15:10:05 33600 +0100 - ovs-node-1 - 22280/140086533916416 - lib/scheduled tasks - 1828 - INFO - Arakoon collapse finished
```

### Test results

Logging previously was Collapsing node: 0lyCLx1w0S2Nn2Sn while now I found Collapsing cluster ovsdb on 10.100.199.152 completed.
Code change has been reflected.
